staging/src/k8s.io/apiserver/pkg/tracing/config.go

```diff
@@ -93,23 +94,26 @@ func validateSamplingRate(rate int32, fldPath *field.Path) field.ErrorList {
 
 func validateEndpoint(endpoint string, fldPath *field.Path) field.ErrorList {
 	errs := field.ErrorList{}
-	if url, err := url.Parse(endpoint); err != nil {
+	if !strings.Contains(endpoint, "//") {
+		endpoint = "dns:///" + endpoint
```

Comment:
just "dns://", right? in this case, we want the string to be parsed as a host by url.Parse, right?

Reply:
fixed